src/design-tokens/tier-2-usage/typography-usage.css

```diff
@@ -90,11 +90,15 @@
 }
 
 @define-mixin eds-theme-typography-button-sm $important {
-   @mixin eds-typography-preset-8 $important;
+   @mixin eds-typography-preset-8-bold $important;
+}
+
+@define-mixin eds-theme-typography-button-md $important {
+   @mixin eds-typography-preset-8-bold $important;
```

Comment:
Medium and small buttons use the same typography; the difference between them is in the padding and height.